frontend/packages/dev-console/src/components/monitoring/overview/MonitoringMetrics.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import * as _ from 'lodash';
 import { requirePrometheus } from '@console/internal/components/graphs';
 import { topWorkloadMetricsQueries } from '../queries';
-import MonitoringDashboardGraph from '../dashboard/MonitoringDashboardGraph';
+import ConnectedMonitoringDashboardGraph from '../dashboard/MonitoringDashboardGraph';
```

Comment:
There's no need to change all these from `MonitoringDashboardGraph` to `ConnectedMonitoringDashboardGraph`. Infact you shouldn't care. You should use the default import as is.

I suggest changing all these import name changes back to simply `MonitoringDashboardGraph`

Reply:
Ok so you'll need to change the name because of the eslint rule for using the same name as an exported value.